api/parts/mgmt/cms.js

```diff
@@ -0,0 +1,204 @@
+                console.log(error);
+                callback(error, null);
+            });
+    }
+
+    fetchPage(1);
+}
+
+/**
+* Transform and store CMS entries in DB
+* @param {params} params - params object
+* @param {array} data - data array
+* @param {function} callback - callback function
+**/
+function transformAndStoreData(params, data, callback) {
+    let lu = Date.now();
+    if (!data || data.length === 0) {
+        //Add meta entry
+        common.db.collection('cms_cache').insertOne({_id: `${params.qstring._id}_meta`, lu}, function() {
+            callback(null, []);
+        });
+    }
+    else {
+        var transformedData = [];
+        for (var i = 0; i < data.length; i++) {
+            transformedData.push(Object.assign({_id: `${params.qstring._id}_${data[i].id}`, lu}, data[i].attributes));
+        }
+        //Delete old entries
+        common.db.collection('cms_cache').deleteMany({'_id': {'$regex': `^${params.qstring._id}`}}, function(err) {
+            if (err) {
+                callback(err, null);
+            }
+            //Insert new entries
+            common.db.collection('cms_cache').insert(transformedData, {ordered: false}, function(err1, entries) {
+                if (err1) {
+                    callback(err1, null);
+                }
+                if (entries) {
+                    //Add meta entry
+                    common.db.collection('cms_cache').insertOne({_id: `${params.qstring._id}_meta`, lu}, function() {
+                        callback(null, entries.ops);
+                    });
+                }
+            });
+        });
+    }
+}
+
+/**
+* Get entries for a given API ID from Countly CMS
+* @param {params} params - params object
+**/
+function syncCMSDataToDB(params) {
+    fetchFromCMS(params, function(err, results) {
+        if (err) {
+            common.returnMessage(params, 500, 'An error occured while fetching entries from CMS: ' + err);
+            return false;
+        }
+        if (results) {
+            transformAndStoreData(params, results, function(err1, storedResults) {
+                if (err1) {
+                    common.returnMessage(params, 500, 'An error occured while storing entries in DB: ' + err1);
+                    return false;
+                }
+                if (storedResults) {
+                    common.returnOutput(params, storedResults);
+                    return true;
+                }
+            });
+        }
+    });
+}
+
+/**
+* Get entries for a given API ID
+* @param {params} params - params object
+* @returns {boolean} true
+**/
+cmsApi.getEntries = function(params) {
+
+    if (!params.qstring._id || AVAILABLE_API_IDS.indexOf(params.qstring._id) === -1) {
+        common.returnMessage(params, 400, 'Missing or incorrect API _id parameter');
+        return false;
+    }
+
+    //WIP
+    var query = {'$or': [ {'_id': {'$regex': `^${params.qstring._id}`}}, {'_id': `${params.qstring._id}_meta`} ]};
```

Comment:
query can be (If you don't have any additional query)
query  = {"_id":{"$regex":"^"+params.qstring._id+"_"}}
because  {Some_ID}_meta also is matched by criteria.

If you have additional query, then it is:

query  = {"_id":{"$regex":"^"+params.qstring._id+"_"},{"$or":[{Passed rules in params.qstring.query}, {'_id': `${params.qstring._id}_meta`} ]}